2.6.md

```diff
@@ -772,10 +772,15 @@ There are also several subordinate objects that provide detailed data to potenti
     <td>string array</td>
     <td>Allowed list of languages for creatives using IETF BCP 47I. Omission implies no specific restrictions, but buyers would be advised to consider language attribute in the <code>Device</code> and/or <code>Content</code> objects if available. Only one of <code>wlang</code> or <code>wlangb</code> should be present.</td>
   </tr>
+  <tr>
+    <td><code>acat</code></td>
+    <td>string array</td>
+    <td>Allowed advertiser categories using the specified category taxonomy. <br>The taxonomy to be used is defined by the <code>cattax</code> field. If no <code>cattax</code> field is supplied IAB Content Taxonomy 1.0 is assumed. Only one of <code>acat</code> or <code>bcat</code> should be present.</td>
```

Comment:
are there known examples of the array of acats being shorter than the array of bcats?